src/watch.ts

```diff
@@ -42,7 +47,11 @@ export const watchCommand = command({
 			runProcess.kill();
 		}
 
-		process.stdout.write(clearScreen);
+		if (!options.noClearScreen) {
+			process.stdout.write(clearScreen);
+		} else {
+			console.log('🔄 tsx: restarting');
```

Comment:
It might be cool to make `noClearScreen` a boolean or string to provide a custom message here.